packages/client/hmi-client/src/components/responsive-matrix/matrix.vue

```diff
@@ -103,6 +103,10 @@ import matrixGridFS from './matrix-grid.fs.glsl';
 
 // EMPTY_POINT used as a fallback value
 const EMPTY_POINT = new Point();
+enum CursorModes {
```

Comment:
we got a type file for the responsive stuff at '@/types/ResponsiveMatrix', this should probably go in there?